ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml

```diff
@@ -3,6 +3,7 @@ chain:
   steps:
   - chain: cucushift-installer-check-cluster-health
   - ref: idp-htpasswd
+  - ref: idp-openldap
```

Comment:
Here an issue comes to my mind what if when more IDPs are implemented. Will we append all of them here? This seems to sound not. We may need to use one IDP for one profile to cover. Maybe we need to use an env var to be passed to specify the IDP type? I don't have mature thought yet. We can discuss next week.

Reply:
As discussed in the meeting, I have made the updates. Please take a look when you have time.